config/settings.yml

```diff
@@ -23,6 +23,8 @@
         :values:
           - machine
     :ignore_terminated_instances: true
+  :s3:
+    :inventory_object_refresh: true
```

Comment:
Old way of inventory refresh is not able to handle large amount of objects. 
For example,  100k S3 objects handled by:
- 2 minutes with the new inventory object refresher
- about 2 hours with the old refresher
As far as removing old refresher is the question of time, we've decided to set the default setting to new mechanism usage.